test/Removal.t.sol

```diff
@@ -1332,30 +1332,6 @@ contract Removal_safeTransferFrom_reverts_ForbiddenTransfer is
   }
 }
 
-contract Removal_safeTransferFrom_reverts_when_paused is UpgradeableMarket {
-  uint256[] private _removalIds;
-
-  function setUp() external {
-    _removalIds = _seedRemovals({
-      to: _namedAccounts.supplier,
-      count: 1,
-      list: false
-    });
-    _removal.pause();
-  }
-
-  function test() external {
-    vm.expectRevert("Pausable: paused");
```

Comment:
I remove this test because we can't actually test it with `expectRevert` anymore now that we are relying on `_beforeTokenTransfer` to enforce the pause on the contract.  This is because `expectRevert` expects the *next* call to revert, but since the next immediate call no longer reverts and instead waits until `_beforeTokenTransfer` gets called to revert, `vm.expectRevert` doesn't register the revert.  This test is covered by the test for pausability on `_beforeTokenTransfer` directly

Reply:
bummer. Probably should be rewritten to instead have this test just expect that  safeTransferFrom invokes whenNotPaused (no idea what that might look like though I'd imagine there's a cheat code for checking which state was accessed)